cmake/utils.cmake

```diff
@@ -263,3 +263,55 @@ function(set_python_module_property target)
         )
     endif()
 endfunction()
+
+# This fuction will populate "out_var" with the default values external
+# projects are expected to need.
+# It can take an optional argument that will replace the list separator
+# in CMake values. For instance, if an external project uses a different
+# list separator, the values in here must be changed to reflect this.
+function(get_external_project_default_values out_var)
```

Comment:
Basically copied this function from Maya for how it setup the external projects.